EmailValidator/Validation/DNSCheckValidation.php

```diff
@@ -21,7 +21,7 @@ class DNSCheckValidation implements EmailValidation
     
     public function __construct()
     {
-        if (!extension_loaded('intl')) {
+        if (!function_exists('idn_to_ascii')) {
```

Comment:
By adding Symfony's polyfill, this shouldn't be needed any more. Since it is a required dep and that dep resolves the intl requirement. However, the Spoofchecker issue (see below) still required intl.

Reply:
rendered moot by #231